src/main/java/edu/illinois/library/cantaloupe/ApplicationServer.java

```diff
@@ -281,6 +284,15 @@ public void start() throws Exception {
                 connector.setIdleTimeout(IDLE_TIMEOUT);
                 server.addConnector(connector);
             }
+
+            // If the Cantaloupe server is started with jmxremote, add the Jetty
+            // jmx extensions to the MBeanServer
+            if (System.getProperties().containsKey(REMOTE_JMX_PORT_PARAMETER) &&
+                    !"".equals(System.getProperty(REMOTE_JMX_PORT_PARAMETER))) {
```

Comment:
Can you explain this condition to me? Does this mean that the server would have to be started with `-Dcom.sun.management.jmxremote.port` set to an empty value? I see that there are some other `com.sun.management.jmxremote.*` keys available, are those not needed or does MBeanContainer read them itself?

Reply:
Sorry, I missed the "!". 🤦‍♂